src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -72,6 +71,7 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
         |master
         |  hotfix/add-trigger""".stripMargin
     )
+    project.openGitMacheteTab()
```

Comment:
Actually... now after a consideration, I think this still isn't a proper solution :/

I'd rather refine the condition for autodiscover to either:
* **not** run the autodiscover when .git/machete is **empty** (only run when it's **missing**)
* or in case the file is empty, check its modification date and refrain from autodiscover if it's modified too recently

Reply:
Still, lemme smuggle the current workaround of yours onto #1167 since I'm tired of re-running the pipelines :trollface:

Reply:
IIRC we had some issue related to sliding out all branches - I do not remember what was the exact problem and how it behaves now. I worry that even though the first option seems to be more reasonable there will be some complicating edge case. anyway I think the effort is not significant and we can try it

Reply:
> we had some issue related to sliding out all branches

Hmmm I think changing the logic to _run autodiscover only when .git/machete is missing, not when empty_ will actually fix the problem with autodiscover happening automatically when someone slides out all branches (which is extremaly unlikely IRL) ;p coz then autodiscover simply won't run, .git/machete will remain empty